### PR TITLE
8.0 social theme

### PR DESCRIPTION
**Pull request by ghost**
Adds a theme module that changes the odoo colors, brand and drilldown button

### CODE REVIEW

File: web_theme_social/static/src/css/theme.css [hunk 1]
```diff
@@ -0,0 +1,731 @@
...
+border-radius: 5px;
+}
+
+
+
+
+
+
+
+
```
Comment (legalsylvain) [inline web_theme_social/static/src/css/theme.css]: some unnecessary empty lines.

File: web_theme_social/static/src/css/theme.css [hunk 2]
```diff
@@ -0,0 +1,731 @@
...
+
+
```
Comment (legalsylvain) [inline web_theme_social/static/src/css/theme.css]: idem

File: web_theme_social/__openerp__.py [hunk 3]
```diff
@@ -0,0 +1,24 @@
+{
+'name': 'Odoo.Social Theme',
+'author': 'Rossa S.A.',
+'category': 'Hidden',
+'version': '8.0.1.0',
+'description':
+"""
+Odoo.Social web theme
+========================
+This modules changes the web interface colors and and brand
```
Comment (legalsylvain) [inline web_theme_social/__openerp__.py]: duplicated word : "and".

File: web_theme_social/__openerp__.py [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+],
+'data' : [
+	'theme.xml',
+	'views/webclient_templates.xml'],
+'qweb' : [
+    "static/src/xml/*.xml",
+],
+'test': [],
+'installable': True,
+'auto_install': True,
```
Comment (legalsylvain) [inline web_theme_social/__openerp__.py]: please, set auto_install to False. 

Or explain why you think it's necessary :
With this setting any people that use OCA / Web will have this module installed. Is it expected ?

**Comment by legalsylvain** [comment]
Hi rossara, 

Thanks for your contribution.

About the code review : 
- Some inline comments ;
- Please remove the two files : 
  1. web_theme_social/static/src/img/favicon.ico
  2. web_theme_social/static/src/img/logo.png
     (promotion by company logo are not allowed in OCA modules) 

About Test : 

I find the display is worth with this module. Did I miss something ? 

**without the module**
before

**with the module**
after

More generally I don't understand the "social" aspect of this module. 

Thanks for your answer.
Regards.

**Comment by loxamir** [comment]
Thanks for the review legalsylvain! i will correct everything and try  again

File: web_theme_social/__openerp__.py [hunk 5]
```diff
@@ -0,0 +1,24 @@
+{
```
Comment (elicoidal) [inline web_theme_social/__openerp__.py]: Please use simple headers in all py files following OCA templates
https://github.com/OCA/maintainer-tools/blob/master/template/module/__openerp__.py#L3

File: web_theme_social/__openerp__.py [hunk 6]
```diff
@@ -0,0 +1,24 @@
...
+'name': 'Odoo.Social Theme',
+'author': 'Rossa S.A.',
+'category': 'Hidden',
+'version': '8.0.1.0',
+'description':
```
Comment (elicoidal) [inline web_theme_social/__openerp__.py]: remove description and create aproper README file following: https://github.com/OCA/maintainer-tools/blob/master/template/module/README.rst

File: web_theme_social/__openerp__.py [hunk 7]
```diff
@@ -0,0 +1,24 @@
...
+'depends': [
+	'web',
+],
+'data' : [
+	'theme.xml',
+	'views/webclient_templates.xml'],
+'qweb' : [
+    "static/src/xml/*.xml",
+],
+'test': [],
```
Comment (elicoidal) [inline web_theme_social/__openerp__.py]: remove empty keys

File: web_theme_social/__openerp__.py [hunk 8]
```diff
@@ -0,0 +1,24 @@
+{
+'name': 'Odoo.Social Theme',
+'author': 'Rossa S.A.',
+'category': 'Hidden',
+'version': '8.0.1.0',
+'description':
+"""
+Odoo.Social web theme
```
Comment (elicoidal) [inline web_theme_social/__openerp__.py]: remove "odoo."

**Comment by rossasa** [comment]
Thanks for all reviews, I solve all the issues you have presented, please check if it's ok now

**Comment by rossasa** [comment]
It's done from my part

**Comment by hbrunn** [comment]
I think the module's name is too generic, call it `theme_social` or something else starting with `theme_`.
Then, you should replace what you do with the announcements with a dependency to `disable_openerp_online` and the branding part with `support_branding`, possibly with this addon bringing some defaults for the parameters that control branding.

**Comment by pedrobaeza** [comment]
There are no news from the author since long, so I close.

**Pull request closed without merging**